README.md

```
@@ -213,8 +215,7 @@ The above .NET nanoFramework interpreter builds include support for the class li
     * [TI CC1352R1_LAUNCHXL](targets/TI_SimpleLink/TI_CC1352R1_LAUNCHXL)
     * [TI CC3220SF_LAUNCHXL](targets/TI_SimpleLink/TI_CC3220SF_LAUNCHXL)
   * Target reference for Raspberry Pi (ChibiOS)
-    * [RP_PICO_RP2040](targets/ChibiOS/RP_PICO_RP2040)
-    * [RP_PICO_W_RP2040](targets/ChibiOS/RP_PICO_W_RP2040)
+    * [RP_PICO_RP2040](targets/ChibiOS/RP_PICO_RP2040)        * [RP_PICO2_RP2350](targets/ChibiOS/RP_PICO2_RP2350)    * [RP_PICO_W_RP2040](targets/ChibiOS/RP_PICO_W_RP2040)
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)